docker-compose.yml

```diff
@@ -17,6 +17,9 @@ sales:
     image: autopilotpattern/workshop-sales
     links:
       - consul
+    environment:
+      - CONSUL_HOST=consul
```

Comment:
With Triton CNS we can get rid of the `links` field and replace this line with an `_env` file that has something like `CONSUL_HOST=consul.svc.${TRITON_ACCOUNT}.${TRITON_DC}.cns.joyent.com`. Given that Consulite has a safe default value maybe we should go:

```yaml
...
env_file: _env
environment:
  - CONSUL_HOST
  - CONSUL_PORT=8500
...
```

This way if the user's environment has a `CONSUL_HOST` field it'll automatically be passed-thru.